docs/production-deployment/cloud/billing-and-cost.mdx

```diff
@@ -0,0 +1,77 @@
+---
+id: billing-and-cost
+title: Billing and Cost
+sidebar_label: Billing and cost
+sidebar_position: 3
+description: As an Account Owner, you can access and manage billing details anytime, with invoices available for download on the Billing page. The typical billing cycle begins on the first of the month (UTC).
+slug: /cloud/billing-and-cost
+toc_max_heading_level: 4
+keywords:
+  - billing
+  - invoices
+  - cost management
+  - account owner
+  - billing cycle
+  - payment options
+  - credit usage
+  - temporal cloud
+  - account balance
+  - invoice history
+tags:
+  - billing
+  - invoices
+  - cost management
+  - account owner
+  - billing cycle
+  - payment options
+  - credit usage
+  - temporal cloud
+  - account balance
+  - invoice history
+---
+
+The Account Owner holds the ability to access and view detailed billing and cost information at any time, ensuring full transparency and control over financial aspects. If you need to assess your spending patterns, check invoice histories, or handle payment details, Temporal Cloud's Cost and Billing capabilities are able to provide the information you need.
+
+## About the Billing page
+
+The **Billing** page includes the following information:
+
+- Your current balance
+- Recent bill
+- Invoice history
+- (If applicable) Credit Grants
+- (If applicable) Credit Purchase
+- (If applicable) Credit Usage
+- (If applicable) Credits Remaining
+
+If you are not on a standard plan, the Billing page may not include all of the information stated above. 
+
+## How to navigate to your invoices
+
+To navigate to your invoices, go to **Settings > Billing**
+
+## Current balance
```

Comment:
I'm not super convinced that each of these needs to be it's own H2 header. 

The could be written like this so the page flows better and so it's easier for readers to know what this page is about. 

"## Understanding the Billing Page:

**Current Balance:** This section shows the amount accrued during the current billing cycle. It includes the date of the latest update and will vary based on your usage. This value also matches the first entry in the Invoices table.

Billing cycles typically start on the first day of the month (UTC). For new accounts, the minimum support fee is prorated based on the sign up date.

**Recent Bill:** This displays your most recent bill amount. If you're set up to pay via Stripe, a **Pay Now** button will be available, directing you to Stripe’s portal to complete payment. If your account is on auto-payment, no action is necessary, but manual payment remains an option if needed.

**Invoices Table:** This table provides a detailed breakdown of your invoices, including:
- **Date (UTC)**: The date range for each invoice.
- **Type:** The invoice type and whether it is for credit purchase or cloud usage.
- **Credits Purchase:** The amount of credits added to your account.
- **Credits Purchased Amount:** The amount you paid for the credits.
- **Credits Usage:** The amount of credits used during the billing cycle.
- **Subtotal:** The total invoice amount before credits are applied.
- **Total:** The remaining balance after credits have been applied."